localisation/00_zuwc_united_localisation.csv

```diff
@@ -3041,6 +3041,30 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
 nick_the_thunderous;the Thunderous;;;;;;;;;;;;;x
 nick_the_lightning_fist;the Lightning-Fist;;;;;;;;;;;;;x
 DEATH_WANDERING_ISLE_NO_KILLER;went to the Wandering Isle;;;;;;;;;;;;;x
+EVTOPTA_MNM_8001_red_crane;The Temple of the Red Crane in Krasarang.;;;;;;;;;;;;;x
+EVTOPTB_MNM_8001_niuzao;Niuzao Temple in Townlong.;;;;;;;;;;;;;x
+EVTOPTC_MNM_8001_white_tiger;The Temple of the White Tiger in Kun-Lai.;;;;;;;;;;;;;x
+EVTOPTA_MNM_8002_jade_serpent;The Temple of the Jade Serpent in the Jade Forest.;;;;;;;;;;;;;x
+CHRONICLE_PILGRIMAGE_TO_RED_CRANE;[Root.GetTitledName] made a pilgrimage to the Temple of the Red Crane.;;;;;;;;;;;;;x
+CHRONICLE_PILGRIMAGE_TO_NIUZAO;[Root.GetTitledName] made a pilgrimage to Niuzao Temple.;;;;;;;;;;;;;x
+CHRONICLE_PILGRIMAGE_TO_WHITE_TIGER;[Root.GetTitledName] made a pilgrimage to the Temple of the White Tiger.;;;;;;;;;;;;;x
+CHRONICLE_PILGRIMAGE_TO_JADE_SERPENT;[Root.GetTitledName] made a pilgrimage to the Temple of the Jade Serpent.;;;;;;;;;;;;;x
+EVTDESC_MNM_8004_red_crane;I have packed my things and made preparations for my household to make do without me. It is time to start my pilgrimage for the Temple of the Red Crane.;;;;;;;;;;;;;x
+EVTDESC_MNM_8005_niuzao;I have packed my things and made preparations for my household to make do without me. It is time to start my pilgrimage for Niuzao Temple.;;;;;;;;;;;;;x
+EVTDESC_MNM_8006_white_tiger;I have packed my things and made preparations for my household to make do without me. It is time to start my pilgrimage for the Temple of the White Tiger.;;;;;;;;;;;;;x
+EVTDESC_MNM_8007_jade_serpent;I have packed my things and made preparations for my household to make do without me. It is time to start my pilgrimage for the Temple of the Jade Serpent.;;;;;;;;;;;;;x
+EVTNAME_MNM_8050_red_crane;The Temple of the Red Crane;;;;;;;;;;;;;x
+EVTDESC_MNM_8050_red_crane;You have finally arrived at the heart of the Krasarang Wilds. One of the August Celestials, Chi-Ji the Red Crane, lives here, in his temple in the middle of a swamp. His followers call him the Spirit of Hope and tell stories of days gone by.\n\nWhen Emperor Shaohao began his journey across Pandaria at some point the emperor fell to a marsh. Without hope, Chi-Ji flew above him and made him realize that he needs to hope once again to save himself. The Red Crane teaches not to lose hope even in the most desperate places like the Krasarang Wilds themselves.;;;;;;;;;;;;;x
```

Comment:
Changed `Without hope` to `Having lost his hope`.
```suggestion
EVTDESC_MNM_8050_red_crane;You have finally arrived at the heart of the Krasarang Wilds. One of the August Celestials, Chi-Ji the Red Crane, lives here, in his temple in the middle of a swamp. His followers call him the Spirit of Hope and tell stories of days gone by.\n\nWhen Emperor Shaohao began his journey across Pandaria at some point the emperor fell into a marsh. Having lost his hope, Chi-Ji flew above him and made him realize that he needs to hope once again to save himself. The Red Crane teaches not to lose hope even in the most desperate places like the Krasarang Wilds themselves.;;;;;;;;;;;;;x
```

Reply:
changed